pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [build-system]
 requires = ["setuptools>=73.0.1", "wheel"]
-build-backend = "setuptools.build_meta"
+build-backend = "setuptools.build_meta:__legacy__"
```

Comment:
What's up with this?

Reply:
This is to make the C++ parser build work with `uv sync` (and also `pip install`). Without this we have to add an extra `uv pip install`.